wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -151,16 +151,18 @@ cuML relies on `clang-format` to enforce code style across all C++ and CUDA sour
 The reasons behind these deviations from the Google style guide are given in comments [here](../../cpp/.clang-format).
 
 ### How is the check done?
-[run-clang-format.py](../../cpp/scripts/run-clang-format.py) is run first by `make`. This script runs clang-format only on modified files. An error is raised if the code diverges from the format suggested by clang-format, and the build fails.
+All formatting checks are done by this python script: [run-clang-format.py](../../cpp/scripts/run-clang-format.py) which is effectively a wrapper over `clang-format`. An error is raised if the code diverges from the format suggested by clang-format. It is expected that the developers run this script to detect and fix formatting violations before creating PR.
```

Comment:
I would append: "To run the script, go to the root of the cuML repository and issue:
`python cpp/scripts/run-clang-format.py`"

Even though the screencast later explains this, I prefer to have the command line written here too.

Reply:
Makes sense, Tamas. I have updated the document accordingly. Please check now.